.github/workflows/pr-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish docker image pre-release
+on:
+  workflow_dispatch:
+defaults:
+  run:
+    shell: pwsh
+env:
+  ci_pipeline: true
+  NHSX_CONTAINER_REGISTRY_NAME: nhsxacrdev # TEMP manual one
+  NHSX_ARM_CLIENT_ID: 43d62150-8efc-4021-898a-37e97ca251d4
+  NHSX_ARM_CLIENT_SECRET: ${{ secrets.NHSX_ARM_CLIENT_SECRET }}
+  NHSX_ARM_TENANT_ID: 812a9cc3-ca27-47d3-b17e-f6dc36887ff3
+  NHSX_ARM_SUBSCRIPTION_ID: 37d5047d-d868-4a63-b09a-38d07d41a889
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set version and tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v5.6
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          tag_prefix: "" # Remove any prefix to ensure compatibility for all build types
+          pre_release_branches: ^((?!main).)*$
```

Comment:
Production branch is `master`. Pre-release branch (for the `development` environment) is development.

Reply:
See comment above around relationship between feature branch, PR releases and merging into the production branch